proposal/main.tex

```diff
@@ -19,15 +19,25 @@
 \maketitle
 
 \section*{General Problem Description}
-Description goes here...
+Aphids are among the most destructive insect pests, and monitoring aphid populations provides important data related to integrated pest management. Inefficient and labor-intensive is the manual identification of aphids. In order to reduce aphid crop damage, recent advances in computer vision algorithms - including object detection - can improve the process of identifying, classifying, and monitoring aphids.
 
 \section*{Required Implementation}
-Implementation stuff goes here...
+For the purpose of identifying the aphids, we will use YOLO/RCNN with preprocessing (fracturing the images and choosing the most likely fragments based on the percentage of green pixels). Our work on improving the baseline results includes adding our own modifications (for example, adding more contrast to the colors in the image) to further enhance the results. We will then evaluate our modified algorithms based on the chosen evaluation metrics (described in the following section). We will simplify the problem by converting it to binary: 1 for aphids and 0 for no aphids.
 
 \section*{General Description of Experiments}
-Experiments go here...
+\begin{enumerate}
+  \item Dataset preprocessing:
+  \begin{enumerate}
+    \item <fill here when the data will be published> 
+  \end{enumerate}
+  \item Randomly split the dataset into train and test sets, 80-20 percent respectively.
+  \item Tune hyperparameter using K-Fold cross validation.
```

Comment:
```suggestion
  \item Tune hyperparameter using K-Fold cross-validation.
```
Unless I'm missing something, K-Fold is not a method for hyperparameter tuning...
It should be combined with a search algorithm that runs the cross-validation with many hyperparameters.

Reply:
K-fold can be used for hyperparameter tuning

look here for example : https://medium.datadriveninvestor.com/k-fold-cross-validation-for-parameter-tuning-75b6cb3214f

Reply:
Ok, looks good.